iOS/UI/Shared/ReplySummaryView.swift

```diff
@@ -79,6 +79,7 @@ class ReplySummaryView: BaseView {
 
         if let existing = self.controller,
             existing.messageId == controller.messageId,
+           controller.hasLoadedAllPreviousReplies,
```

Comment:
I needed to add this unfortunately as sometimes the reply you just added would not show right away.